### PR TITLE
Better translations of spanish language

### PR DESCRIPTION
**Pull request by ajtudela**

### CODE REVIEW

File: PKHeX/Resources/text/es/lang_es.txt [hunk 1]
```diff
@@ -628,158 +628,158 @@ B_Cancel = Cancelar
...
+L_Decoration = Decor.:
+L_Rotation = Rotación:
+L_X = Coord. X:
+L_Y = Coord. Y:
 --
-f_PKM = Trainer Pokémon
+f_PKM = Entrenador Pokémon
 GB_PKM = PKM
-L_Participant = Participant:
+L_Participant = Participant:e
```
Comment (evandixon) [inline PKHeX/Resources/text/es/lang_es.txt]: Did you mean to add the "e" after the colon?

**Comment by evandixon** [comment]
Nitpick: The end of Dulzura is cut off by a pixel or two.  It may or may not be worth adding some extra spacing between these labels and textboxes.
image

Other than that and my previous comment, looks good.

File: PKHeX/Resources/text/es/lang_es.txt (hunk 1 above)
Comment (ajtudela) [inline PKHeX/Resources/text/es/lang_es.txt]: Sorry, It is "L_Participant = Participante:"

Comment (kwsch) [inline PKHeX/Resources/text/es/lang_es.txt]: No, it's fine. I'll rectify it in a later commit.

**Pull request merged**